open-weather-map-api/src/dependencies/cleaning/cleaning.py

```diff
@@ -0,0 +1,88 @@
+import os
+import pandas as pd
+import country_converter as coco
+import pycountry
+import math
+from datetime import datetime
+
+DATA_DIR = '../scraping/weather_forecast.csv'
+
+class Cleaner:
+
+	def __init__(self):
+		return None
+	
+	def load_data(self):
+		"""load csv data from the directory"""
+		
+		df = pd.read_csv(DATA_DIR)
+		return df
+	
+	def remove_duplicates(self,df):
+		"""remove duplicates row in the datframe"""
+				
+		df.drop_duplicates(keep='first', inplace=True)
+		return df
+		
+	def check_missing_values(self,df):
+		"""check for missing values replace it with NaN"""
+		df.replace(' ', math.nan)
+		return df
+
+	def country_iso_code(self,df):
+		"""convert country codes to ISO 3166-1 alpha3"""
+		
+		df['country'] = df['country'].apply(lambda x : pycountry.countries.get(alpha_2=x).alpha_3 if (len(x) == 2) else pycountry.countries.get(name=x).alpha_3)
```

Comment:
Please use code-formatter extensions and libraries available in VSCODE to format such long lines

Reply:
thanks !! I will do it.